packages/blockchain-wallet-v4-frontend/utils/capitalize.js

```diff
@@ -0,0 +1,3 @@
+export default function capitalize (string) {
```

Comment:
can you just add this function to `services/validationHelper.ts` it has related formatting things in there already

Reply:
also rename it to `capitalizeFirstLetter`

Reply:
Ok will do